app_logging.py

```diff
@@ -28,3 +28,8 @@ def createLog(module):
     logger.addHandler(consoleLog)
 
     return logger
+
+
+stage = os.getenv('STAGE', 'qa')
+
+logger = create_logger(f'drb-etl-pipeline-{stage}')
```

Comment:
What is the purpose of this stage variable?

Reply:
Just to differentiate between the prod and qa logger in New Relic. They are differentiated by the application name but good to have different logger names as well.

Reply:
That makes sense and a nice clarity check.